test/e2e/runners/ethereum_multiparty_test.go

```diff
@@ -27,6 +27,7 @@ func TestEthereumMultipartyE2ESuite(t *testing.T) {
 	suite.Run(t, new(multiparty.IdentityTestSuite))
 	suite.Run(t, new(multiparty.OnChainOffChainTestSuite))
 	suite.Run(t, new(multiparty.TokensTestSuite))
+	suite.Run(t, new(multiparty.TokensRemoteNameTestSuite))
```

Comment:
When I set "suite" I actually meant "runner". I think this test should be in its own runner since it makes changes to the token plugin config.

Reply:
I suppose "suite" was the first step though.